src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/dashboard/dashboard.component.html

```diff
@@ -1,6 +1,8 @@
 <div>
   <cd-refresh-selector></cd-refresh-selector>
 
+  <cd-motd></cd-motd>
```

Comment:
This is by intention. I decided to implement the MOTD as a dashboard widget to do not waste parts of the UI with this message. Due the fact that the dashboard is the entry point of the UI and the MOTD is the top most widget everybody should realize it. Another point was to do not have to take care about having to manage the visibility of the banner. If a banner would have been used, as a user i would like to be able to hide it somehow to give me back the wasted view height. After a page refresh i expect that the banner is still hidden; so this needs to be managed via cookie which finally increases the complexity of this feature. Because of this a widget is the better solution IMO.

Reply:
IMHO if the purpose of this feature is to get noticed by all users, since the announcements might be important:
* **this should be visible from 'all corners' of the Dashboard**, hence the landing page widget is not enough. However, a banner or a toast notification would work.
* **this should appear "instantaneously"**. What if the user doesn't reload the landing page (or other pages) for a while (since the session cookie has a 24 hour expiration)?

Regarding the persistence, as you have defined 3 severity levels, this might behave accordingly:
* "danger" message cannot be hidden,
* "warning" can be hidden but reappear after a while (till next page reload: this doesn't require any cookie at all or it can be a sessionStorage persistence) 
* "info" can behave like the Telemetry one (cookie or localStorage based) 

Additionally, the landing page has been crafted to display dynamically-changing key cluster information, and to fit vertically into a Full HD screen. Putting it into the Landing page requires too much space for a piece of information that barely changes over time. 

What about something like this:

![image](https://user-images.githubusercontent.com/37327689/124638055-ceb8c200-de8a-11eb-8ebd-2d54ac92e900.png)

Reply:
I'm not happy with the banner approach because there are huge problems with the current implementation of the `password change` and `telemetry` banner incl. the navigation component which is responsible for rendering the banners, navigation and content area. The banners are limited to a height of 38px and the navigation component is responsible to calculate the position/height of the banners. That's bad code, the notifications banners and all other containing components are responsible to calc their size themself. I tried to refactor the navigation component by using @angular/flex-layout component, but it comes out that this is too much for this PR. The whole navigation component needs to be refactored to get rid of this inelegant code.

I will open a bug report that will show the problems.

Reply:
I've opened a tracker issue: https://tracker.ceph.com/issues/51575